script/deploy/single/05b_DeployMultiChainLayerZeroTellerWithMultiAssetSupport.s.sol

```diff
@@ -49,6 +63,46 @@ contract DeployMultiChainLayerZeroTellerWithMultiAssetSupport is BaseScript {
         );
         require(address(teller.endpoint()) == config.lzEndpoint, "OP Teller must have messenger set");
 
+        // check if the DVN is configured and print a message to the screen to inform the deployer if not.
         return address(teller);
     }
+
+    function _checkUlnConfig(ConfigReader.Config memory config) internal {
+        ILayerZeroEndpointV2 endpoint = ILayerZeroEndpointV2(config.lzEndpoint);
+        address lib = endpoint.defaultSendLibrary(config.peerEid);
+        bytes memory configBytes = endpoint.getConfig(config.teller, lib, config.peerEid, 2);
+        UlnConfig memory ulnConfig = abi.decode(configBytes, (UlnConfig));
+
+        if (ulnConfig.confirmations == 0) {
+            _setConfig(endpoint, lib, config);
+            return;
+        }
+        uint8 numRequiredDVN = ulnConfig.requiredDVNCount;
+        uint8 numOptionalDVN = ulnConfig.optionalDVNCount;
+        for (uint256 i; i < numRequiredDVN; ++i) {
+            if (ulnConfig.requiredDVNs[i] == dead) {
+                _setConfig(endpoint, lib, config);
+                return;
+            }
+        }
+        for (uint256 i; i < numRequiredDVN; ++i) {
```

Comment:
is this supposed to be numOptionalDVN?

Reply:
It will have a numOptional,

In what's here I assumed we were ONLY using LZ but we decided on call today to include optionals